deploy/hooks/post-restore

```diff
@@ -8,4 +8,19 @@ if [ -x /etc/init.d/apache2 ]; then
     sudo /etc/init.d/apache2 restart
 fi
 
-exit 0
```

Comment:
I mean if apache doesn't restart after 20 secs you can aready exit earlier

And 70 secs is the worsth case scenario for Varnish to detect if the instance is running when we know for sure apache has started.